### DISCUSSION TITLE
Identify Under Replicated Partition (URP)

### DISCUSSION THREAD

**Discussion by treyhendon** [provectus/kafka-ui#3216] [Q&A] [answered]
In the UI, it's easy to tell that a certain Topic has an URP.  How do we identify the specific Broker that the URP is located?

In a different Kafaka Web UI program, we have columns for Leader, Replicas, and In-Sync Replicas.  Seeing a diff in the two Replica columns allows us to know "this partition is out of sync, reboot the Leader".

In a perfect world, I'd expect to see a "red dot" by the Replica-Broker that is not in-sync when viewing the Topic Overview.

How are others identifying and fixing URPs?

**Comment by Haarolean** [comment, marked as answer]
Done within https://github.com/provectus/kafka-ui/issues/3427#issuecomment-1469958164
